sdk/iot/Azure.Iot.Hub.Service/src/Authentication/SharedAccessSignatureConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Iot.Hub.Service.Authentication
+{
+    internal static class SharedAccessSignatureConstants
+    {
+        public const int MaxKeyNameLength = 256;
```

Comment:
Each of these could use meaningful xml comments as well

Reply:
Tracked here: https://msazure.visualstudio.com/One/_workitems/edit/7462994